src/request/remove-key/remove-key.css

```diff
@@ -37,30 +37,62 @@
     margin-bottom: 0;
 }
 
-.page#privacy:not(.hide-passphrase) .hide-for-passphrase {
+.page#remove-key:not(.show-final-confirm) .final-confirm,
+.page#remove-key.show-final-confirm .hide-for-final-confirm,
+.page#download-key-file.show-download .hide-for-download,
+.page#download-key-file:not(.show-download) .download-key-file,
+.page#download-key-file:not(.show-download) .back-to-remove {
     display: none;
 }
-.page#privacy.hide-passphrase .passphrase-box {
-    display: none;
+
+.page#remove-key .backup-option {
+    display: flex;
+    align-items: center;
+    margin: 4rem 3rem 0;
 }
 
-.page#remove-key.show-passphrase-box .hide-for-passphrase {
-    display: none
+.page#remove-key .backup-option + .backup-option {
+    margin-top: 3rem;
 }
 
-.page#remove-key:not(.show-passphrase-box) .passphrase-box {
-    display: none;
+.page#remove-key .backup-option .login-file-svg-container,
+.page#remove-key .backup-option img {
+    min-width: 8.375rem;
+    height: 14.744rem;
 }
 
-.page#download-key-file.show-download .hide-for-download {
-    display: none
+.page#remove-key .backup-option .login-file-svg-container svg {
+    width: 100%;
+    height: 100%;
 }
 
-.page#download-key-file:not(.show-download) .download-key-file,
-.page#download-key-file:not(.show-download) .back-to-remove {
-    display: none;
+.page#remove-key .backup-option > *:nth-child(2) {
+    margin-left: 3rem;
+}
+
+.page#remove-key .page-body {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
 }
 
-.page-footer {
-    padding: 1rem;
+.page#remove-key .page-footer button {
+    margin-top: 0;
+    width: calc(100% - 7rem);
+}
+
+.page#remove-key.page#remove-key:not(.show-final-confirm) .page-footer #remove-key-label-confirm {
```

Comment:
Double `.page#remove-key` here. And generally very high specificity here. Is that necessary?